modules/bullet/space_bullet.h

```diff
@@ -69,11 +69,9 @@ extern ContactAddedCallback gContactAddedCallback;
 
 class BulletPhysicsDirectSpaceState : public PhysicsDirectSpaceState {
 	GDCLASS(BulletPhysicsDirectSpaceState, PhysicsDirectSpaceState)
-private:
-	SpaceBullet *space;
 
 public:
-	BulletPhysicsDirectSpaceState(SpaceBullet *p_space);
+	SpaceBullet *space;
```

Comment:
Can you please leave this private and add a "friend class SpaceBullet" so it can access to it anyway

Reply:
Done.

FWIW, I only did what GodotPhysics does, but indeed it's at least a bit nicer not to make the variable public.